src/Raven.TestDriver/RavenTestDriver.cs

```diff
@@ -278,6 +280,8 @@ private static IDocumentStore RunServer()
                 Urls = new[] { url.AbsoluteUri }
             };
 
+            store.Conventions.DisableTopologyCache = true;
```

Comment:
inline it?

            var store = new DocumentStore
            {
                Urls = new[] { url.AbsoluteUri },
                Conventions = {
                    DisableTopologyCache  = true
                }
            };

Reply:
done